lib/beaker/logger.rb

```diff
@@ -83,6 +83,25 @@ def initialize(*args)
 
       @destinations = []
 
+      @color_levels = options[:color_levels] || {
```

Comment:
I like the idea here, though you probably want to use a merge, not a wholesale replacement, so that users can partially override.